Common/include/parallelization/mpi_structure.cpp

```diff
@@ -27,9 +27,17 @@
 
 #include "mpi_structure.hpp"
 
+
+/* Initialise the MPI Communicator Rank and Size */
 int CBaseMPIWrapper::Rank = 0;
 int CBaseMPIWrapper::Size = 1;
+
+/* Set the default MPI Communicator */
+#ifdef HAVE_MPI
 CBaseMPIWrapper::Comm CBaseMPIWrapper::currentComm = MPI_COMM_WORLD;
+#else
+CBaseMPIWrapper::Comm CBaseMPIWrapper::currentComm = 0;  // dummy value
+#endif
```

Comment:
I like this solution.
The rank and size are updated by the SetComm function so everything should work ok when creating a driver from scratch.

Reply:
Glad to hear that. Is anything still missing ? Not sure how to go about the failing check for code complexity ...

Reply:
Not for me, I'll ask around the developers meeting tomorrow if anyone else has comments, codefactor glitches out sometimes no need to do anything.

By the way would you like to be added to su2code? You can then have branches here instead of in your fork, makes it simpler to manage future PR's you might open.

Reply:
Yes, I suppose it would be more convenient that way. Thanks !